src/migrations/043_userinfo_oidc.ts

```diff
@@ -0,0 +1,34 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  const result = await knex('changelog').select('id').where({ id: 43 });
+
+  if (result.length) {
+    // Migration has been applied using the old patch system
+    return;
+  }
+
+    await knex.schema.createTable('user_info', table => {
+        table.increments()
+        table.string('sub', 50).notNullable()
+        table.string('name', 200).nullable()
+        table.string('middle_name').nullable()
+        table.string('given_name', 30).nullable()
+        table.string('family_name', 30).nullable()
+        table.date('birthdate').nullable()
+        table.string('picture').nullable()
+        table.integer('updated_at').unsigned().nullable()
+        table.text('address').nullable()
+        table.string('locale', 5).nullable()
+        table.string('zoneinfo').nullable()
```

Comment:
Just a thought, but in many database types we can add comments to fields, like this patch:

https://github.com/curveball/a12n-server/blob/main/src/migrations/20240908210700_block_failed_login_attempts.ts

The benefit of having comments on fields is when we auto-generate the db-types.ts file, these comments also get added to the types like here:

https://github.com/curveball/a12n-server/blob/main/src/db-types.ts#L221

It's not strictly needed so entirely up to you. As you can see in db-types.ts, I only started doing this for later migrations.

Reply:
oh yeah I liked those! 

I was also wondering if we should keep picture as I saw a ton of examples wit hthat

Reply:
I *love* the idea of having a picture, but maybe it's just a separate ticket. It's not really as simple as we need to have a way to actually change the picture via an upload. Uploads can be malicious, and we need to resize probably so there's just a bit more to think about.

I kept out `gender` btw because i would rather use `pronouns` which has the added benefit of also not being usable by the US government.

Reply:
yeah I saw the spec... only male and female 😬

Reply:
I ex_spec_t spec to be stuffy and outdated tho